components/application-mgt/org.wso2.carbon.identity.application.mgt.ui/src/main/resources/web/application/edit-sp-template.jsp

```diff
@@ -77,8 +77,8 @@
         String templateName = "";
         String templateDesc = "";
         try {
-            templateName = request.getParameter("templateName");
-            templateDesc = request.getParameter("templateDesc");
+            templateName = request.getParameter("templateName").trim();
```

Comment:
Use owasp Encoder and encode templateName and templateDesc whenever used in HTML body. Eg https://github.com/wso2/carbon-identity-framework/pull/1849/files#diff-fa021d3fe65d081541f88d70e1381a26R126